components/tag-list/tag-list-item-mixin.js

```diff
@@ -191,11 +191,11 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 
 				announce(message);
 			}
+			this._onFocusIn();
 
 			await this.updateComplete;
 			// delay the focus to allow focusin to fire
 			setTimeout(() => {
-				this._onFocusIn();
```

Comment:
This might break something else I recently worked on but I'll have to refresh my memory on what it was exactly.

Reply:
Sounds good - let me know if you think of it. I only recently added `_onFocusIn` so it might post-date the problems you worked on.

Reply:
All clear. I was mixing this up with the [parent](https://github.com/BrightspaceUI/core/pull/4498/files#diff-2589d7434564f559d1ad28a1e94699c08b1d183ad460f77275d0d6fa1bd903ddR422) `focusIn` I added, which is what the comment is referring too. Maybe we should make that clearer.